tools/topology/topology2/avs-tplg/CMakeLists.txt

```diff
@@ -16,6 +16,15 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-hda-generic-4ch.bin,USE_CHAIN_DMA=true
 # CAVS SDW topology with passthrough pipelines
 "cavs-sdw\;cavs-sdw"
 
+"cavs-sdw\;sof-tgl-rt711-rt1316-rt714\;NUM_SDW_AMPS=2,SDW_DMIC=1"
```

Comment:
@bardliao can we add a comment here distinguishing the 3 topologies below. something along the lines saying " the first 2 are with SDW DMICs and the last one uses the SoC DMIC"

Reply:
> @bardliao can we add a comment here distinguishing the 3 topologies below. something along the lines saying " the first 2 are with SDW DMICs and the last one uses the SoC DMIC"

ack